pallets/investments/src/lib.rs

```diff
@@ -23,7 +23,8 @@ use frame_support::{
 	traits::tokens::fungibles::{Inspect, Mutate, Transfer},
 };
 use frame_system::pallet_prelude::*;
-use sp_runtime::traits::{AccountIdConversion, CheckedSub};
+use sp_runtime::traits::AccountIdConversion;
```

Comment:
Just curious, why we cannot bundle them together?

Reply:
Tested something and forgot to revert, my bad!